test/0.4.24/lido/lido.accounting.test.ts

```diff
@@ -75,6 +86,7 @@ describe("Lido:accounting", () => {
     });
 
     it("Updates beacon stats", async () => {
+      await updateLidoLocatorImplementation(await lido.getLidoLocator(), { accounting: deployer });
```

Comment:
Looks like it's cleaner to impersonate accountOracle contract address in before and connect it as a default one.